src/pages/FarmV4/FarmContainer.tsx

```diff
@@ -21,7 +21,7 @@ export const FarmContainer: FC = () => {
     setPool
   } = useFarmContext()
   const [searchTokens, setSearchTokens] = useState<string>('')
-  const [showDeposited, setShowDeposited] = useState<boolean>(existingUserCache.farm.showDepositedFilter)
+  const [showDeposited, setShowDeposited] = useState<boolean>(userCache.farm.showDepositedFilter)
```

Comment:
This should be:

existingUserCache.gamma.showDeposited filter.

We don't want to update Farm filter here.

It was wrong earlier too.

Reply:
It is not, it is userCache - if you scroll up you will see it imports it from the hook

Reply:
The code previously updated this, was the original code wrong then?

Reply:
Yes, it was wrong. It is the showDeposited filter for Farm (and not ssl). So we just need to update the filter for 'gamma' case & not 'farm' case. It already exists in farmv3. Farmv4 = Gamma related changes.

Reply:
cool luke fixed it